modules/ipi-install-configuring-networks.adoc

```diff
@@ -0,0 +1,9 @@
+// This is included in the following assemblies:
+//
+// ipi-install-prerequisites.adoc
+
+[id='configuring-networks_{context}']
+
+= Configuring Networks
+
+Documentation for customizing the network configuration is out-of-scope for this release. For details on customizing the network configuration, see https://access.redhat.com/solutions/5460671[Customizing network configuration for OCP 4.6 bare metal IPI installation]. 
```

Comment:
link:https://access.redhat.com/solutions/5460671[Customizing network configuration for OCP 4.6 bare metal IPI installation].